packages/integration_test/lib/_callback_io.dart

```diff
@@ -21,7 +21,7 @@ class IOCallbackManager implements CallbackManager {
   @override
   Future<Map<String, dynamic>> callback(
       Map<String, String> params, IntegrationTestResults testRunner) async {
-    final String command = params['command'];
+    final String command = params['command']!;
```

Comment:
Surprised this needs a `!`, since it's a `Map<String, String>`. (I'd expect the `!` if it were a `Map<String, String?>`, but I guess Maps may return null if the key is not found)

Reply:
This confused me too. The `operator[]` is nullable, even if the value type is not nullable.